packages/@sanity/base/src/components/validation/validationList.tsx

```diff
@@ -0,0 +1,81 @@
+import React, {useCallback} from 'react'
+import {
+  ObjectSchemaType,
+  Path,
+  Marker,
+  isValidationErrorMarker,
+  isValidationWarningMarker,
+} from '@sanity/types'
+import styled from 'styled-components'
+import {ListItem} from './listItem'
+
+interface ValidationListProps {
+  documentType?: ObjectSchemaType
+  kind?: 'simple'
+  markers: Marker[]
+  onFocus?: (path: Path) => void
+  onClose?: () => void
+  truncate?: boolean
+}
+
+const ListContainer = styled.div`
+  max-width: 250px;
+  max-height: 60vh;
+  overflow: auto;
+`
```

Comment:
I think we can get rid of `ListContainer` entirely, and let the Popover deal with content overflow. [See example here.](https://sanity-ui-workshop.sanity.build/components/menu/constrained-in-boundary) And [the source code of the example here](https://github.com/sanity-io/design/blob/next/packages/%40sanity/ui/src/components/menu/__workshop__/constrainedInBoundary.tsx).

Reply:
It cuts off the list where the document status bar starts, and it's too wide even with width 0 and 1. Is there a fix for this with that approach?

Reply:
I was looking at the code and I'm not sure if there's anything that has been done about this. Popover seems to be already being used in the `ValidationMenu` component?